src/components/navbar/index.ts

```diff
@@ -33,16 +36,47 @@ import { NavbarRoot, NavbarPrimary, NavbarSecondary } from './Navbar';
  * ```tsx
  * <Navbar>
  *  <Navbar.Primary>
- *      ...
+ *    <Navbar.ItemsContainer>
+ *      <Navbar.Item to="/projects">Projects</Navbar.Item>
+ *      <Navbar.DropdownItem
+ *        items={
+ *          <>
+ *            <DropdownMenu.ItemLink to="/link1"><FormattedMessage id='ext1.name'/></DropdownMenu.ItemLink>
+ *            <DropdownMenu.ItemLink to="/link2"><FormattedMessage id='ext2.name'/></DropdownMenu.ItemLink>
+ *          </>
+ *        }
+ *      ><FormattedMessage id='more'/></Navbar.DropdownItem>
+ *    </Navbar.ItemsContainer>
```

Comment:
[question] Is `FormattedMessage` inside the `</Navbar.DropdownItem>` sensitive to newlines? I find having it on a single line like that harder to read.